app/controllers/clients/files_controller.rb

```diff
@@ -46,32 +46,27 @@ def new
     def create
       @file = file_source.new
       @file.errors.add :tag_list, 'You must specify file contents' if file_params[:tag_list].blank?
-      @file.errors.add :file, 'No uploaded file found' unless file_params[:file]
+      @file.errors.add :client_file, 'No uploaded file found' unless file_params[:client_file]
```

Comment:
Philosophical question -- should we add parens to method calls when we find them like this?
```suggestion
      @file.errors.add(:client_file, 'No uploaded file found') unless file_params[:client_file]
```
And, do/should we have a style convention for it? When there are multiple params like this, I like to see the parens -- but in the `render :new` below, I think `render(:new)` looks *less* readable to me...

Reply:
I agree.  I haven't found the cop for it yet.  Fixed these though.